rest-api/config/base_config.json

```diff
@@ -28,7 +28,8 @@
   ],
   "dna_sample_test_codes": [
     "1ED04",
-    "1SAL"
+    "1SAL",
+    "1SAL2"
```

Comment:
@eweitz FYI, we will need this change reflected in the SQL for the new metrics API.